src/utils/regexp.ts

```diff
@@ -11,3 +11,5 @@ const exact = (exp) => `^${exp}$`;
 export const wsRegexp = new RegExp(exactStart(ws));
 export const dnsPathRegexp = new RegExp(exact(`${dns}${port}?(${segment})*/?`));
 export const ipv4Regexp = new RegExp(exact(`${ipv4}${port}?(${segment})*/?`));
+
+export const syntheticAssetRegexp = new RegExp(exact('0[xX]03[0-9a-fA-F]+'));
```

Comment:
We can add XSTUSD address in this RegExp using OR `|` operand to not think about this asset in components code